Nebraska Lexicon/load_stage.sql

```diff
@@ -1396,7 +1396,11 @@ JOIN concept c1 ON c1.concept_code = cs.concept_code
 JOIN concept_relationship cr ON cr.concept_id_1 = c1.concept_id
 	AND cr.relationship_id = 'Maps to'
 	AND cr.invalid_reason IS NULL
-JOIN concept c2 ON c2.concept_id = cr.concept_id_2;
+JOIN concept c2 ON c2.concept_id = cr.concept_id_2
```

Comment:
would it be better just to move 
DO $_$
BEGIN
	PERFORM VOCABULARY_PACK.CheckReplacementMappings();
END $_$;

--12. Add mapping from deprecated to fresh concepts
DO $_$
BEGIN
	PERFORM VOCABULARY_PACK.AddFreshMAPSTO();
END $_$;

to the end of script to avoid this patch usage